rocky/reports/report_types/dns_report/report.py

```diff
@@ -24,14 +24,15 @@ class DNSReport(Report):
 
     def generate_data(self, input_ooi: str, valid_time: datetime) -> dict[str, Any]:
         ref = Reference.from_str(input_ooi)
-        tree = self.octopoes_api_connector.get_tree(ref, valid_time, depth=3, types={DNSRecord, Finding}).store
+        records_tree = self.octopoes_api_connector.get_tree(ref, valid_time, depth=1, types={DNSRecord}).store
+        findings_tree = self.octopoes_api_connector.get_tree(ref, valid_time, depth=3, types={Finding}).store
 
         findings = []
         finding_types: dict[str, dict] = {}
         records = []
         security = {"spf": True, "dkim": True, "dmarc": True, "dnssec": True, "caa": True}
 
-        for ooi_type, ooi in tree.items():
+        for ooi_type, ooi in findings_tree.items():
             if isinstance(ooi, Finding):
```

Comment:
The `if isinstance` is not necessary anymore.

Reply:
We do since the OOIs in the path to the finding are also in this tree

<img width="1022" alt="image" src="https://github.com/minvws/nl-kat-coordination/assets/43830693/9bab87fe-d444-489d-a624-534e4bbd1fc4">